app/tests/spec/models/auth_brokers/fx-desktop.js

```diff
@@ -156,10 +158,34 @@ define([
           })
           .then(function () {
             assert.equal(data.email, 'testuser@testuser.com');
+            assert.isFalse(data.verified);
             assert.isTrue(data.verifiedCanLinkAccount);
             assert.isTrue(channelMock.send.called);
           });
       });
+
+      it('indicates whether the account is verified', function () {
+        // set account as verified
+        account.set('verified', true);
+
+        var data;
+        sinon.stub(channelMock, 'send', function (message, _data, done) {
+          if (message === 'can_link_account') {
+            return done(null, { data: { ok: true }});
+          } else if (message === 'login') {
+            data = _data;
+            return done(null);
+          }
+        });
+
+        return broker.beforeSignIn('testuser@testuser.com')
+          .then(function () {
+            return broker._notifyRelierOfLogin(account);
+          })
+          .then(function () {
+            assert.isTrue(data.verified);
```

Comment:
There are two reasons I did it this way above:

1. I like all the assertions grouped together. When all the assertions are in reading order, they are easy to find.
1. I saved the data and checked it later because I couldn't figure out how to check only a single item in an object using `channelMock.send.calledWith`. Maybe I just lazily read the sinon docs. Suggestions would be great.